docs/content/docs/dev/datastream/project-configuration.md

```diff
@@ -327,9 +318,8 @@ description = """Flink Quickstart Job"""
 
 ext {
     javaVersion = '1.8'
-    flinkVersion = '1.13-SNAPSHOT'
-    scalaBinaryVersion = '2.11'
-    slf4jVersion = '1.7.32'
+    flinkVersion = '1.14.3'
```

Comment:
Shouldn't this be 1.15-SNAPSHOT in this case?

Reply:
Correct, I'll fix it.

Reply:
done